docs-ref-conceptual/includes/cli-install-linux-apt.md

```diff
@@ -187,18 +189,28 @@ AZ_REPO="bookworm"
 
 ### Elementary OS (EOS) fails to install the Azure CLI
 
-EOS fails to install the Azure CLI because `lsb_release` returns `HERA`, which is the EOS release name.  The solution is to fix the file `/etc/apt/sources.list.d/azure-cli.list` and change `hera main` to `bionic main`.
+EOS fails to install the Azure CLI because `lsb_release` returns `HERA`, which is the EOS release name.  The solution is to fix the file `/etc/apt/sources.list.d/azure-cli.sources` and change `Suites: hera` to `Suites: bionic`.
```

Comment:
Actually, I am not sure if we really support Elementary OS.

Reply:
No idea, but the section was there.

Reply:
A fun fact, Element OS patches python-apt to use upstream LSB codename when `add-apt-repository`: https://github.com/elementary/os-patches/issues/136#issuecomment-858145591

Reply:
Tbh, I didn't know Element OS existed or had such a big user base justifying a separate section before reading the installation docs.